Tasks/DotNetCoreCLIV2/Tests/L0.ts

```diff
@@ -13,6 +13,7 @@ describe('DotNetCoreExe Suite', function () {
     });
 
     it('fails if the dotnet tool is not found', (done: MochaDone) => {
+        this.timeout(5000);
```

Comment:
is this required?

Reply:
yes this particular test was failing.In existing test cases also we have mentioned timeout of 1000 took reference from it